### PR TITLE
Update Box Button Style and move Icons to the Right

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2021-11-08 at 3 37 54 PM

Screen Shot 2021-11-08 at 3 37 42 PM

Screen Shot 2021-11-08 at 3 54 45 PM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx [hunk 1]
```diff
@@ -438,14 +438,15 @@ const ListBuckets = ({
                         <br />
                         <br />
                         To get started,&nbsp;
-                        <button
+                        <a
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx:441]: This needs to be a button as anchor requires an href per ESLint rules

File: portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx [hunk 2]
```diff
@@ -438,14 +438,15 @@ const ListBuckets = ({
...
                           className={classes.link}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx:442]: Text size must be added in link class

File: portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx [hunk 3]
```diff
@@ -300,14 +300,14 @@ const ListTenants = ({ classes, setErrorSnackMessage }: ITenantsList) => {
                               <br />
                               <br />
                               To get started,&nbsp;
-                              <button
+                              <a
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx:303]: Same case with this anchor

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx:441]: this causes the underlying text to look small o_o

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by Alevsk — APPROVED** [review]

**Pull request merged**